lib/util.js

```diff
@@ -236,6 +236,7 @@ exports.getVideoID = function(link) {
   if (!id) {
     return new Error('No video id found: ' + link);
   }
+  id = id.substring(0,11);
```

Comment:
Thanks for the PR!

Could you add a space after the comma? Trying to keep the styling consistent across files.